python/ray/train/lightning/lightning_trainer.py

```diff
@@ -308,6 +309,17 @@ def __init__(
         preprocessor: Optional[Preprocessor] = None,
         resume_from_checkpoint: Optional[Checkpoint] = None,
     ):
+        run_config = run_config or RunConfig()
+        lightning_config = lightning_config or LightningConfigBuilder().build()
```

Comment:
wait, do we want people to pass in a dict or the raw Builder()?
the type annotation says it's a raw dict already.